main/streams/xp_socket.c

```diff
@@ -719,9 +748,27 @@ static inline int php_tcp_sockop_bind(php_stream *stream, php_netstream_data_t *
 	}
 #endif
 
+#ifdef SO_LINGER
+	if (PHP_STREAM_CONTEXT(stream)
+		&& (tmpzval = php_stream_context_get_option(PHP_STREAM_CONTEXT(stream), "socket", "so_linger")) != NULL) {
```

Comment:
It's already part of socket option so there is no point for so_ prefix.

Reply:
well I added it to have consistency but I do not mind removing it